packages/pds/src/config/config.ts

```diff
@@ -175,10 +175,20 @@ export const envToCfg = (env: ServerEnvironment): ServerConfig => {
   const bskyAppViewCfg: ServerConfig['bskyAppView'] = {
     url: env.bskyAppViewUrl,
     did: env.bskyAppViewDid,
-    proxyModeration: env.bskyAppViewModeration ?? false,
     cdnUrlPattern: env.bskyAppViewCdnUrlPattern,
   }
 
+  if (!env.modServiceUrl) {
+    throw new Error('Must configure PDS_MOD_SERVICE_URL')
+  } else if (!env.modServiceDid) {
+    throw new Error('Must configure PDS_MOD_SERVICE_DID')
+  }
+
+  const modServiceCfg: ServerConfig['modService'] = {
+    url: env.modServiceUrl,
+    did: env.modServiceDid,
+  }
```

Comment:
Ahhh, I bet we'll have to get this going on the entryway.  Maybe time to properly land that work...

Reply:
yeahh we should check in on how exactly that'll function